src/main/java/codechicken/nei/config/IMCHandler.java

```diff
@@ -126,32 +145,56 @@ private static void handleRegisterCatalystInfo(IMCMessage message) {
         }
 
         if (!processedCatalystSenders.contains(message.getSender())) {
-            NEIClientConfig.logger.info("Processing registerCatalystInfo from " + message.getSender());
             processedCatalystSenders.add(message.getSender());
         }
         final NBTTagCompound tag = message.getNBTValue();
-        final String handlerID = tag.getString("handlerID");
-        if (handlerID.isEmpty()) {
-            NEIClientConfig.logger.warn("Missing handlerID for registerCatalystInfo!");
+        final String catalystHandlerID;
+        if (tag.hasKey("handlerID")) {
+            NEIClientConfig.logger.warn(
+                    "Received tag 'handlerID' for registerCatalystInfo IMC from {}, this is deprecated, please use 'catalystHandlerID' instead",
```

Comment:
revert this frivolous change

Reply:
I have been trying to reduce potential confusion from using our APIs. Unfortunately, `handlerID` here can be confused with the old `handler` field in the register handler info IMC, despite the former allowing for a wider range of potential values (I've documented the details in the README), that's why I have decided to rename it.

BTW, I'm working on updating the Et Futurum GTNH NEI handlers to fix some of the issues with the integration and to use as a case study.

Reply:
I should make the log message clearer that it's just a renaming.

Reply:
in this very moment, you are Ϡ

Reply:
The message says it's "deprecated", does that mean that the old name will stop working in a future version? That seems like a bad idea for maintaining compatibility with mods that may not be updated in the future.
And if the old name won't stop working in a future version, is there a reason to log a warning at all?